inbm/integration-reloaded/test/ucc/UCC_GOOD_COMMAND_TEST.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+set -euxo pipefail # DO NOT REMOVE -- used to fail test if intermediate command fails
+
+source /scripts/test_util.sh
+
+start_time=$(get_time)
+print_all_error() {
+   echo "TEST FAILED!!!"
+}
+trap print_all_error ERR
+
+
+UCC_MSG='{"some": "arbitrary", "json": "string"}'
+
+test_echo Triggering UCC Command Test
+listen_ucc_command
+
+trigger_ucc_command "${UCC_MSG}"
+sleep 2
+if grep -q "arbitrary" ucc-command-response; then
```

Comment:
This is incorrect--should check command response is exactly equal to UCC_MSG.  Also should check for the OK response that we return to the UCC server.